internal/analytics/events.go

```diff
@@ -42,8 +45,36 @@ func MockedTracker(name string, action string, flags []string) *MockTracker {
 			"id":      id,
 			"name":    name,
 		},
-	}
-	tracker := MockTracker{ID: id}
-	tracker.On("SendEvent", mockedTrackingArgs...)
+	}...)
+	tracker.On("SendEvent", []interface{}{
+		"testAccessToken",
+		"http://test.com",
+		"CLI Command Completed",
+		map[string]interface{}{
+			"id":      id,
+			"outcome": outcome,
+		},
+	}...)
 	return &tracker
 }
+
+func SendCommandRunEvent(name string, cmd *cobra.Command, analyticsTracker Tracker) {
+	cmd.Root().Annotations = map[string]string{"tracking": "true"}
+	analyticsTracker.SendEvent(
+		viper.GetString(cliflags.AccessTokenFlag),
+		viper.GetString(cliflags.BaseURIFlag),
+		"CLI Command Run",
+		CmdRunEventProperties(cmd, name),
+	)
+}
+
+func SendCommandCompletedEvent(outcome *string, analyticsTracker Tracker) {
```

Comment:
Why is `outcome` a pointer to a string instead of a string?

Reply:
I changed this. outcome is now just a string